tutorials/02a_RunningREMINDLocally.md

```diff
@@ -1,10 +1,146 @@
+REMIND_repos_dirs="/my/first/path:/my/second/path"
+# On windows, it would look something like
+# REMIND_repos_dirs="C:\Users\<your windows username>\REMIND;D:\REMIND"
+# depending on your used file paths
+```
+
+You do not have to unpack tar-archives as this will be done during the start-up of your REMIND run.
+
+## Start a run
+
+In a terminal session change directory to the location where REMIND was cloned via `cd <REMIND_DIR>` and start a run as you would do on the cluster. As a linux user, type
+
+```bash
+Rscript start.R
+```
+
+or as a Windows user, type
+
+```PowerShell
+Rscript.exe .\start.R
+```
+
+## Known Issues & Further Debugging
+
+### MAGCFG_STORE File Path Resolution Fails in `prepare.R`
+
+*Telltale sign* Function `prepare()` in `prepare.R` is unable to resolve the file path to `MAGCFG_STORE`, ie. the location where MAGICC configurations are stored. The error message reads:
+
+```
+Error in prepare() :
+  ERROR in MAGGICC configuration: Could not find file  ./core/magicc/MAGCFG_STORE/MAGCFG_USER_OLDDEFAULT.CFG
+```
+
+This bug might be specific to Windows systems and is still under investigation. However, there is a
+
+*Solution* Locate lines
+
+```R
+magcfgFile = paste0('./magicc/MAGCFG_STORE/','MAGCFG_USER_',toupper(cfg$gms$cm_magicc_config),'.CFG')
+[...]
+system(paste0('cp ',magcfgFile,' ','./magicc/MAGCFG_USER.CFG'))
+```
+
+in file `<REMIND_DIR>/scripts/start/prepare.R` (around line numbers 680ff). Replace them with
+
+```R
+magcfgFile = normalizePath(paste0('../../core/magicc/MAGCFG_STORE/','MAGCFG_USER_',toupper(cfg$gms$cm_magicc_config),'.CFG'))
+[...]
+system(paste0('cp ',magcfgFile,' ','../../core/magicc/MAGCFG_USER.CFG'))
+```
+
+and then restart your run.
+
+### `renv` Confuses Itself
+
+*Telltale sign* Rscript fails at the beginning of a run printing
+
+```
+Rscript : Error in eval(quote({ : object '..version..' not found
+```
+
+This occurs usually after an initial run has completed and seems to only affect people running REMIND with R-version `4.1.X`. The
+
+*Solution* is to checkout the original version of `<REMIND_DIR>/renv/activate.R` via
+
+```bash
+git checkout ./renv/activate.R
+```
+
+then restart your run.
+
+### Support RSE with Extensive Log Files
+
+In case your REMIND run fails and you wish to be supported by REMIND RSE in getting it to run properly, please help us by recording the logging output of the run. On linux, start your run via
+
+```bash
+Rscript start.R &>> ~/failed_REMIND_run.log
+```
+
+or on Windows via
+
+```PowerShell
+Rscript.exe .\start.R *>> ~/failed_REMIND_run.log
+```
+
+This writes *all* output from your run to a file in your home directory called `failed_REMIND_run.log`. After hitting enter in your terminal session, none of the usual output will show up since it is being redirected into the aforementioned file. You can still follow the progress of your run by opening a second terminal window and type in
+
+```bash
+tail -n 25 -f ~/failed_REMIND_run.log
+```
```

Comment:
Or
```
Rscript start.R 2>&1 | tee ~/failed_REMIND_run.log
```

Reply:
Would this allow for user interaction with the REMIND/Rscript process?

Reply:
Sure does.

Reply:
I refer to a scenario in which the user has to select the run type by putting in a number. `tee` would not relay the user input to `Rscript` and the run would get stuck. But that scenario might not even happen with running REMIND on a local machine, so I'm happy with the `tee` based solution as well

Reply:
Why should `tee` relay the input?  It has not been redirected.

Reply:
Exactly. Thus I recommend to use two separate terminal sessions instead of piping the output to `tee`